credoai/reporting/dataset_base.py

```diff
@@ -159,7 +199,7 @@ def create_report(self, filename=None):
         ref_name = ref.iloc[0]["feature"]
         ref_type = ref.iloc[0]["feature type"].split("_")[0]
 
-        fig = plt.figure(figsize=(8, 4), dpi=150)
+        fig = plt.figure(dpi=150)
```

Comment:
I saw you setting dpi, and thought that made sense. So I added a "get_style" function that I started using in the nlp report. We can standardize our plotting later. See [here](https://github.com/credo-ai/credoai_lens/blob/ian/nlp_report_improv/credoai/reporting/nlp_generator.py#L51)